src/components/Select.jsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { testBoard } from '../letters/Testboard';
+
+const SelectNumber = ({
+  numberSelected,
+  setNumberSelected,
+  setSelectNumber,
+  wordLengths,
+  setWordLengths,
+  squares,
+  setSquares,
+  wordNo,
+  setWordNo,
+  setErrorMessage,
+}) => {
+  const lengthList = [2, 3, 4, 5];
+  let useTestBoard = false;
```

Comment:
This can probably be just changed to a constant outside the render function

Reply:
These are both const now and moved to a global scope.